### PR TITLE
[4.0] RavenDB-11642 - Setup wizard on macOS issues

### PR DESCRIPTION
**Pull request by grisha-kotler**
http://issues.hibernatingrhinos.com/issue/RavenDB-11642

### CODE REVIEW

File: src/Raven.Server/Utils/Cli/BrowserHelper.cs [hunk 1]
```diff
@@ -6,22 +6,31 @@ namespace Raven.Server.Utils.Cli
...
                     Process.Start("xdg-open", url);
                 }
+
+                error = null;
+                return true;
             }
             catch (Exception e)
             {
                 Console.WriteLine("Could not start browser: " + e.Message);
+                error = e.Message;
```
Comment (ayende) [inline src/Raven.Server/Utils/Cli/BrowserHelper.cs]: better to use the full text, no?

File: src/Raven.Server/ServerWide/SecretProtection.cs [hunk 2]
```diff
@@ -535,7 +535,8 @@ private static void AddCertificateChainToTheUserCertificateAuthority(X509Certifi
             else
                 collection.Import(rawBytes, password, X509KeyStorageFlags.PersistKeySet | X509KeyStorageFlags.Exportable | X509KeyStorageFlags.MachineKeySet);
 
-            using (var userIntermediateStore = new X509Store(StoreName.CertificateAuthority, StoreLocation.CurrentUser, 
+            var storeName = PlatformDetails.RunningOnMacOsx ? StoreName.My : StoreName.Root;
```
Comment (iftahbe) [inline src/Raven.Server/ServerWide/SecretProtection.cs]: accidentally changed StoreName.CertificateAuthority to StoreName.Root?

File: src/Raven.Server/Utils/Cli/BrowserHelper.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Server/Utils/Cli/BrowserHelper.cs]: fixed

File: src/Raven.Server/ServerWide/SecretProtection.cs (hunk 2 above)
Comment (grisha-kotler) [reply src/Raven.Server/ServerWide/SecretProtection.cs]: fixed

**Pull request merged**